doc/computation.rst

```diff
@@ -185,7 +185,7 @@ windowed rolling, convolution, short-time FFT etc.
 
 Because the ``DataArray`` given by ``r.construct('window_dim')`` is a view
 of the original array, it is memory efficient.
-One can also use ``construct`` to compute a weighted mean:
+You can also use ``construct`` to compute a weighted mean:
```

Comment:
I do think this should explicitly say `weighted rolling mean`, may have got lost from https://github.com/pydata/xarray/pull/2012#discussion_r176850037